src/components/answersList/styles.css

```diff
@@ -0,0 +1,36 @@
+.answers-list-table {
+  display: block;
+  table-layout: fixed;
+  width: 90%;
+  margin: 0 auto 30px auto;
+  height: 15rem;
+  overflow-y: scroll;
+  letter-spacing: 1px;
+}
+
+.answers-list-container div {
+  padding: 0;
+}
+
+.answers-header {
+  text-align: left;
+  color: lightgray;
+  width: 100%;
+}
+
+.answers-list-tbody {
+  width: 100%;
+}
+
+.answers-list-thead {
+  width: 100%;
+  position: sticky;
+  background-color: white;
+  top: 0;
```

Comment:
When I scroll the table, a thin line of answers is visible above the thead:
![Screenshot from 2022-01-01 17-16-45](https://user-images.githubusercontent.com/47178494/147854943-2acdfe74-0504-440b-8860-387f84375035.png)
You could set `top` to `-1px` to prevent it.

Reply:
I've added `border-spacing: 0` like you said above, and I think it worked for this issue too.

Reply:
Unfortunately I still have the problem. I'm testing it on Chrome, version 96.